app.js

```diff
@@ -207,6 +217,7 @@ webmakerProxy(app, middleware.checkForAuth);
 // Main page
 app.get('/',
         middleware.setNewPageOperation,
+        middleware.decryptToken,
```

Comment:
good middleware design would require something like:
```
app.get('/',
  middleware.setNewPageOperation,
  middleware.ensureToken,
  middleware.decryptToken,
  ...
```
So that you separate concerns: one function to do one, and only one, thing.